requirements.txt

```diff
@@ -122,7 +122,7 @@ pycparser==2.19
 pycryptodome==3.9.0
 pycryptodomex==3.9.0
 PyGithub==1.43.8
-# gpgme-devel not available on centos8 pygpgme==0.3
+pygpgme==0.3
```

Comment:
pygpgme does not work with current versions of gpgme. Use `python3-gpg` instead from EL8 repos. They should be visible as Python package `gpg` with pip, but only installable via RPM.